controllers/operatorpolicy_controller.go

```diff
@@ -603,6 +635,12 @@ func buildSubscription(
 	subscription.ObjectMeta.Namespace = ns
 	subscription.Spec = spec
 
+	// If the policy is in `enforce` mode and the allowed CSVs are restricted,
+	// the InstallPlanApproval will be set to Manual so that upgrades can be controlled.
+	if policy.Spec.RemediationAction.IsEnforce() && len(policy.Spec.Versions) > 0 {
+		subscription.Spec.InstallPlanApproval = operatorv1alpha1.ApprovalManual
```

Comment:
Do you need to set an `else` here to set automatic in case `policy.Spec.Versions` gets unset later on?

Reply:
If the policy is edited and `policy.Spec.Versions` is removed, then it will just use whatever is in `policy.Spec.Subscription`... in that case either Manual or Automatic is valid.

When a user makes an enforced policy specifying versions, we're interpreting that as them wanting "automatic" updates. And so it would make sense if they set Automatic in the subscription spec. But from OLM's perspective, it needs to be Manual, so that the operator policy controller can approve specific versions (which it will do autonomously).

Reply:
Right. Also, from a quick test on a cluster, it looks like the CRD doesn't require it to be set (even though its description says it's required...) and it seems to default to `Automatic` behavior when it isn't specified.